src/cli.rs

```diff
@@ -0,0 +1,49 @@
+use crate::data;
+use clap::{Parser, Subcommand};
+use std::path::PathBuf;
+use std::{env, io};
+
+#[derive(Debug, Parser)]
+#[clap(name = "rustig")]
+#[clap(about = "A git implemented in RUST", long_about = None)]
+struct Cli {
+    #[clap(subcommand)]
+    command: Commands,
+}
+
+#[derive(Debug, Subcommand)]
+enum Commands {
+    #[clap()]
+    Init {},
```

Comment:
Question: quelle est la bonne pratique en Rust ? avec ou sans accolade pour un enum sans argument
```suggestion
    Init,
```

Reply:
À vérifier avec rustfmt et le compiler. mais je ne vois pas de contre indication à enlever les accolades